otelib/backends/services/client.py

```diff
@@ -0,0 +1,93 @@
+"""Client for services backend."""
+from otelib.backends.services import (
+    DataResource,
+    Filter,
+    Function,
+    Mapping,
+    Transformation,
+)
+
+
+# pylint: disable=duplicate-code
+class OTEServiceClient:
```

Comment:
Same comments here as for the Python client class; we should have an abstract base class for the client.

Reply:
Agreed, but think we should make this a separate issue, see my comments above